src/components/Calculator.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Calculator.css';
+
+function Button(props) {
+  const { value } = props;
+
+  // Define button types based on their values
+  let type = '';
+  if (['+', 'x', '-', '÷'].includes(value)) {
+    type = 'operation';
+  } else if (!Number.isNaN(Number(value))) {
+    type = 'number';
+  } else {
+    type = 'function';
+  }
+
+  return <button type="button" data-type={type} data-value={value}>{value}</button>;
+}
+
+Button.propTypes = {
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]).isRequired,
+};
```

Comment:
- [ ] Kindly make sure to add the code for the `Button` component in a separate file and import it to the `Calculator` component instead

Reply:
hi @Ismailco but why should I do that? that's not on the requirements, Kindly add some feedback to understand why are you asking me this...

Reply:
Hi @rudicarrilloypr, it is required to use 1 component inside the `Calculator`, also it's best practice to divide your code in React, one file should include one component

Reply:
- [ ] Well resolved 🥇